### DISCUSSION TITLE
Participatory Mapping & Critical Cartography

### DISCUSSION THREAD

**Discussion by anmindi** [albertkun/21S-ASIAAM-191A#74] [Past Thinking Caps]
I was not familiar with the idea of participatory mapping until this course, but I am familiar with community-based participatory research (CBPR)---and the ways in which it has become simplified and misconstrued from its original principles. Phrases like "community-based" and "participatory" have become quite popularized but can also be used in a way that is merely philanthropic and still exploitative of minoritized peoples, extracting local knowledge and lacking reciprocity & respect. Our last reading reminded me of a case study I had to read for one of my public health classes about a global health intervention in Bangladesh during the early 2000's, led by Columbia University. Due to a prior intervention in the 1960's, led by UNICEF, Bangladesh's well water became polluted with arsenic, leading to the largest mass poisoning in recorded history. A team of American epidemiologists, geologists, etc. led an observational study to look at the health effects of arsenic, which reflects a larger history of American interventions and medical racism. Because of funding constraints, more effort was focused on _examining_ health effects rather than on trying to implement _solutions_ to prevent arsenic poisoning (you do not need empirical data to know that arsenic poisoning is bad for your health). It also struck me how their target locations were chosen based on factors like convenience (distance from the airport) rather than on community need/assessment. Furthermore, when discussing such case studies today, not enough emphasis is placed on the reason behind occurrences of global emergencies. It seems that many natural disasters are merely presented as natural, rather than as being provoked by Western exploits. Although interventions for drilling well water seem benevolent, would an intervention have been needed in the first place if colonizing countries had not been extracting their natural resources? All of this is to say that participatory mapping (and research methodologies in general) can be improved by leading with community interest/need over institutional interest and questioning institutional & imperial systems.

**Comment by epham36** [comment]
Thank you for this comment! As someone who wants to promote health equity in the field of public health, it is so important to recognize the strong biases in the institutions that are purported to "serve the greater good," knowing their own agendas and their own complicity in creating health inequities and disparities. This definitely brings to mind critiques I have read about the non-profit industrial complex, in which nonprofit organizations, which are often perceived and self-promoted to be social justice oriented, are confined by the agendas of their funding sources in their ability to provide for the common good (Samimi, 2010). Where programs that address the root cause of health issues and accurately assess community needs are critical to address poor health outcomes at their cause, many nonprofits, research institutions, and governmental bodies provide "band-aid services that are unable to exact real change focused on jusice for all." To fail to recognize the true causality of arsenic poisoning and other environmental health hazards is to overlook the discriminatory systems in place -- that are often upheld by the very institutions that purport to address these health issues.
